### PR TITLE
Hook up rates api

### PR DESCRIPTION
**Pull request by hatched**

### CODE REVIEW

**Comment by hatched** [comment]
#### To QA

- Use the `pay` flag with guiproxy on `staging`.
- open `deployment-flow.js`
- add {this._generatePricingSection()} to the `render` method after `{this._generateModelNameSection()}
- load the gui
- add an app and open the deployment flow
- You should see the xhr request to pull in the rates, and those rates in the UI

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/juju-gui/8644/

File: jujugui/static/gui/src/app/components/deployment-flow/deployment-flow.js [hunk 1]
```diff
@@ -1177,7 +1177,7 @@ DeploymentFlow.propTypes = {
   getDiagramURL: PropTypes.func,
   getEntity: PropTypes.func,
   getGithubSSHKeys: PropTypes.func.isRequired,
-  getSLAMachineRates: PropTypes.func,
+  getSLAMachineRates: PropTypes.func.isRequired,
```
[the remark this answers is not in the document]
Comment (hatched) [reply jujugui/static/gui/src/app/components/deployment-flow/deployment-flow.js]: Good catch, fixed!

**Comment by hatched** [comment]
Thanks @huwshimi :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/juju-gui/8645/

**Comment by jujugui** [comment]
Build failed: Attempt to land pull request failed
        build url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge/2785

**Pull request merged**